app/vmalert/main.go

```diff
@@ -76,13 +80,15 @@ func main() {
 		logger.Fatalf("can not get external url: %s ", err)
 	}
 	notifier.InitTemplateFunc(eu)
```

Comment:
Should this be actioned after new URL was generated? I wonder how functions like `pathEscape` would behave with new flag set. Btw, can we cover it with tests as well?

Reply:
i added the test, i don't think we need to re-init template funcs